io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -146,12 +160,8 @@ def test_jbod(self):
         self.check_pass(cmd, "Failed to set JBOD on")
         cmd = "%s /c%d show jbod" % (self.tool, self.controller)
         self.check_pass(cmd, "Failed to show the JBOD status")
-        cmd = "%s /c%d/eall/sall set jbod" % (self.tool, self.controller)
-        self.check_pass(cmd, "Failed to convert the drives to JBOD")
         cmd = "%s /c%d show " % (self.tool, self.controller)
         self.check_pass(cmd, "Failed to show the adapter details")
-        cmd = "%s /c0/eall/sall set good force" % self.tool
```

Comment:
This test should enable jbod mode, and disable while exiting.

Reply:
hmmm No it would still break the code flow.

JBOD=on is used indirectly to clear any leftovers from the RAID creation before running an scenario.

JBOD=off is needed before running RAID level tests.

Eventhough on an outlook it looks as if we really need to do that.  But for the code flow perspective depending on what state the system is this would be essential.